applications/GPIO/flip_store/manifest.yml

```diff
@@ -0,0 +1,20 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/jblanked/FlipStore.git
+    commit_sha: b6e11fd53c8427d7f8345355e16e609e161e65f7
+short_description: "Download apps via WiFi directly to your Flipper Zero."
+description: "@./assets/README.md"
```

Comment:
I'd like to clarify some things before we merge this PR.

I've peeked into your code, I see you are using your own API to list and download apps.
Is it somehow uses official catalog API? If not then you explicitly should state that. 
Also I don't see how you are handling different API versions, are you always compiling agains latest release/dev?
Folder that you are using for the installation is inside of the Apps folder, which is system folder. I kinda do understand why you are using it, but it also conflicts with our app delivery system.

Please move this flip_store into separate PR and let's have more detailed discussion